juice/src/layers/common/rnn.rs

```diff
@@ -0,0 +1,524 @@
+//! Create a Recursive Layer
+//!
+//! TODO: Add Docs
```

Comment:
;) a reference to wikipedia and maybe a few lines of what RNN are would suffice for now